wfe/web-front-end.go

```diff
@@ -1045,7 +1044,7 @@ func (wfe *WebFrontEndImpl) BuildID(response http.ResponseWriter, request *http.
 
 	response.Header().Set("Content-Type", "text/plain")
 	response.WriteHeader(http.StatusOK)
-	detailsString := fmt.Sprintf("Boulder=(%s %s) Golang=(%s) BuildHost=(%s)", core.GetBuildID(), core.GetBuildTime(), runtime.Version(), core.GetBuildHost())
+	detailsString := fmt.Sprintf("Boulder=(%s %s)", core.GetBuildID(), core.GetBuildTime())
```

Comment:
I was going to just totally filter out access to the /build API URL at the HTTPS endpoint, actually.

It's up to you whether you want this information for development builds, because I'm comfortable leaving this right as it is and filtering.